packages/react/src/create-strict-api/__tests__/generics.test.tsx

```diff
@@ -188,21 +195,31 @@ describe('createStrictAPI()', () => {
   describe('type success', () => {
     it('should pass type check for css()', () => {
       const styles = css({
+        // @ts-expect-error — should be a value from the schema
+        padding: '10px',
```

Comment:
Padding is now typed in the example schema above https://github.com/atlassian-labs/compiled/pull/1636/files#diff-2b3750c8f7eefa0e3b3f59fd1e2c8b4f84d0ce0997ba9c24a60287424fa6d89eR14